test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,117 @@
+const agent = require('superagent-promise')(require('superagent'), Promise);
+const { expect, assert } = require('chai');
+const md5 = require('md5');
+
+const urlBase = 'https://api.github.com';
+
+describe('Given a user logged in github', () => {
+  const username = 'aperdomob';
```

Comment:
You use two users in the tests.  does it work with the same token?

Reply:
we need only logged one time